jekyll/_cci2/browser-testing.md

```diff
@@ -92,74 +92,40 @@ jobs:
 ## Sauce Labs
 {: #sauce-labs }
 
-Sauce Labs operates browsers on a network that is separate from CircleCI build containers. To allow the browsers access
-the web application you want to test, run Selenium WebDriver tests with Sauce Labs on CircleCI using Sauce Labs' secure tunnel [Sauce Connect](https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy).
+Sauce Labs has an extensive network of operating system and browser combinations you can test your web application against. Sauce Labs supports automated web app testing using Selenium WebDriver scripts as well as through `saucectl`, their test orchestrator CLI, which can be used to execute tests directly from a variety of JavaScript frameworks.
 
-Sauce Connect allows you to run a test server within the CircleCI build container and expose it (using a URL like `localhost:8080`) to Sauce Labs' browsers. If you run your browser tests after deploying to a publicly accessible staging environment, you can use Sauce Labs in the usual way without worrying about Sauce Connect.
+### saucectl
+{: #saucectl }
 
-This example `config.yml` file shows how to run browser tests through Sauce Labs against a test server running within a CircleCI build container.
+If you are using JavaScript to test your web application, you can still take advantage of the Sauce Labs platform by using [`saucectl`](https://docs.saucelabs.com/testrunner-toolkit) with the JS framework of your choice, and then integrating the [saucectl-run orb](https://circleci.com/developer/orbs/orb/saucelabs/saucectl-run) in your CircleCI workflow.
+
+1. Add your `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` as [environment variables]({{site.baseurl}}/2.0/env-vars/) in your Circle CI project.
+2. Modify your CircleCI project `config.yml` to include the saucectl-run orb and then call the orb as a job in your workflow.
 
 {% raw %}
 ```yaml
-version: 2
+version: 2.1
+orbs:
+  saucectl: saucelabs/saucectl-run@1.0.0
```

Comment:
Update to 1.1.0 when this orb update goes live: https://github.com/saucelabs/saucectl-run-orb/pull/3#pullrequestreview-770322950

Reply:
Thanks, @KyleTryon. Any estimates on when that will happen?

Reply:
Okay. I have tee'd up the change in anticipation of the orb update.

Reply:
@sweeneyskirt-sl I'll ping you when it's available !
 It will be version 2.0.0 as we introduced a breaking change.